galley/pkg/config/analysis/analyzers/deprecation/deprecation.go

```diff
@@ -60,29 +60,6 @@ func (*FieldAnalyzer) analyzeSidecar(r *resource.Instance, ctx analysis.Context)
 
 	sc := r.Message.(*v1alpha3.Sidecar)
 
-	if sc.Localhost != nil {
```

Comment:
Can you explain why these messages are being removed?  I don't see any API changes to sidecar as a part of this PR, but maybe I'm missing something?

Reply:
From https://github.com/istio/api/pull/1529

The API was merged but never implemented or shipped to users (always hide from docs + unimplemented)

Reply:
@nrjpoddar knows better. I'm copying from https://github.com/istio/istio/pull/25585